server/bleep/src/webserver/answer.rs

```diff
@@ -146,7 +152,17 @@ pub async fn handle(
             .await
             .map_err(super::internal_error)?;
 
-        let grown_text = grow(&doc, relevant_snippet, 60);
+        let mut grow_size = 40;
```

Comment:
this change allows for us to grow the snippet dynamically until we reach 2000 tokens or 200 lines, which ever comes first.

the reason for setting the 200 line limit is to prevent small files (<100) lines from causing this to loop infinitely.